content/en/docs/contribute/_index.md

```diff
@@ -28,41 +28,62 @@ Kubernetes documentation welcomes improvements from all contributors, new and ex
 
 ## Getting started
 
-Anyone can open an issue about documentation, or contribute a change with a pull request (PR) to the [`kubernetes/website` GitHub repository](https://github.com/kubernetes/website). You need to be comfortable with [git](https://git-scm.com/) and [GitHub](https://lab.github.com/) to operate effectively in the Kubernetes community.
+Anyone can open an issue about documentation, or contribute a change with a
+pull request (PR) to the
+[`kubernetes/website` GitHub repository](https://github.com/kubernetes/website).
+You need to be comfortable with
+[git](https://git-scm.com/) and
+[GitHub](https://lab.github.com/)
+to operate effectively in the Kubernetes community.
```

Comment:
nit: could remove `to operate effectively in the Kubernetes community.`. I am not a big fan of the word `operate`.

Reply:
I'm gonna change the word to `work`. Please let me know if it's okay.